protocols/autonat/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.13.0
+
+- Due to the refactor of `Transport` it's no longer required to create a seperate transport for
+AutoNAT where port reuse is disabled. This information is now passed by the behaviour.
+  See [PR 4568](https://github.com/libp2p/rust-libp2p/pull/4568).
+
+<!-- Update to libp2p-swarm v0.45.0 -->
```

Comment:
You mean the comment? Personally, I don't think it adds any value to users of `rust-libp2p` because they very likely only depend on the `libp2p` crate so it just clutters the changelog. It is there to "please the tools" which I also recognise is not ideal.

Up to you how you want to handle this in the future, just wanted to add some context! :)

Reply:
thanks for jumping in and clarify Thomas! So, my idea is that it's clear that the version was update to a new minor due to an update of a exposed dependency, or to accomodate a new minor update of the `libp2p`.